core/dbt/events/types.py

```diff
@@ -4,8 +4,8 @@
 from dbt.events.stubs import (
     _CachedRelation,
     BaseRelation,
-    ParsedModelNode,
     ParsedHookNode,
+    ParsedModelNode,
```

Comment:
The only conflict I fixed myself with the rebase was removing `_ReferenceKey` from here since @nathaniel-may added it above in his PR